src/platform_impl/ios/app_state.rs

```diff
@@ -792,8 +792,8 @@ pub unsafe fn handle_main_events_cleared() {
     if !this.has_launched() {
         return;
     }
-    match this.state_mut() {
-        &mut AppStateImpl::ProcessingEvents { .. } => {}
+    match *this.state_mut() {
+        AppStateImpl::ProcessingEvents { .. } => {}
```

Comment:
Don't think you need either of these, thanks to auto-ref these days?

Reply:
I'm not sure, there must be a reason why clippy suggested it.